apidef/adapter/gqlengineadapter/v2/adapter_proxy_only.go

```diff
@@ -0,0 +1,95 @@
+package v2
```

Comment:
the pakage name v2 is very confusing. I do understand that graphql-go-tools/v2 is meant, but any other developer will be very confused.

It will probably make sense to call this enginev3 to bring it inline with our naming, because the engine version does not care if it is graphql-go-tools/v1 or v2 under the hood.

Reply:
yeah i had this worry too, but figured i'll align with reviewing. I will unify the naming so it's more straightforward 🦾